tests/e2e/specs/in-game/sevens.spec.js

```diff
@@ -1487,4 +1487,44 @@ describe('Playing sevens at the end of the deck', () => {
       scrap: [Card.SEVEN_OF_CLUBS],
     });
   });
+
+  it('Cannot play one-off if deck is empty', () => {
+    cy.setupGameAsP1();
+    cy.loadGameFixture(1, {
+      p0Hand: [],
+      p0Points: [],
+      p0FaceCards: [],
+      p1Hand: [Card.SEVEN_OF_CLUBS, Card.ACE_OF_CLUBS],
+      p1Points: [],
+      p1FaceCards: [],
+      topCard: Card.SIX_OF_HEARTS,
+      secondCard: Card.SEVEN_OF_HEARTS,
+      deck: [Card.FIVE_OF_DIAMONDS],
+    });
+    cy.drawCardOpponent();
+    cy.get('#deck').click();
+    cy.drawCardOpponent();
+    cy.get('[data-player-hand-card=7-0]').click();
+    cy.get('[data-move-choice=oneOff').should('have.class', 'v-card--disabled');
+  });
+
+  it('Cannot play one-off on last card (7) in deck after playing 7 one off - special case', () => {
+    cy.setupGameAsP1();
+    cy.loadGameFixture(1, {
+      p0Hand: [],
+      p0Points: [],
+      p0FaceCards: [],
+      p1Hand: [Card.SEVEN_OF_CLUBS, Card.ACE_OF_CLUBS],
+      p1Points: [],
+      p1FaceCards: [],
+      topCard: Card.SIX_OF_HEARTS,
+      secondCard: Card.SEVEN_OF_HEARTS,
+      deck: [],
+    });
+    cy.drawCardOpponent();
+    cy.get('#deck').click();
+    cy.passOpponent();
+    cy.get('[data-player-hand-card=7-0]').click();
+    cy.get('[data-move-choice=oneOff').should('have.class', 'v-card--disabled');
```

Comment:
This case seems really similar to the above one, where there are no cards in the deck and therefore we can't play a 7. Based on the logic you updated in the `MoveChoiceCard`, I expected this test case to be the case where the top card of the deck is a 7 and there's no second card, with a 7 in the player's hand to confirm that they can play the 7 in their hand, but that the 7 on the deck then can't be played as it's one-off (because there's no second card)